server/events/pull_updater.go

```diff
@@ -41,3 +47,50 @@ func (c *PullUpdater) updatePull(ctx *command.Context, cmd PullCommand, res comm
 		ctx.Log.Err("unable to comment: %s", err)
 	}
 }
+
+type FeatureAwarePullUpdater struct {
```

Comment:
can you separate out code that is lyft specific vs. upstreamable? ie. lyft specific code should go in a subpackage called lyft, or just in the `server/lyft` package.

See how @msarvar does it here:
https://github.com/lyft/atlantis/pull/197/files

Reply:
done